### DISCUSSION TITLE
Export Options for ToonSquid?

### DISCUSSION THREAD

**Discussion by yulrun** [keiwando/toonsquid#445] [General]
Good afternoon,

I'm looking at the potential of using ToonSquid for game art. Basically weighing my options that are available on iPadOS. 

I like that it has bone animations.

Can you export as GIF or SpriteSheet?

**Comment by Cat-Butler** [comment]
Hi @yulrun!
 
Animated GIF is available as an export format.
SpriteSheet's are not currently available, but sequential images are. There are probably iPad apps that can convert those to SpriteSheet format (Pixaki, perhaps?).

Strictly and pedantically speaking, ToonSquid does not (yet) have bones, although the option for a hierarchical setup with custom pivot points amounts to the same thing for animating using forward kinematics.
